### ISSUE TITLE
Stop asking if I want to reset the terminal

### ISSUE THREAD

**Issue by grosch-intl** [git-for-windows/git#4323]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options

git version 2.39.1.windows.1
cpu: x86_64
built from commit: b03dafd9c26b06c92d509a07ab01b01e6d0d85ee
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.19044.2604]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
$ cat /etc/install-options.txt

Editor Option: VIM
Custom Editor Path:
Default Branch Option:
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled
```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

Nope.

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Bash

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
Alt-F8
```
 - What did you expect to occur after running these commands?

The terminal should clear, as it did until recently.

 - What actually happened instead?

A dialog is appearing asking if I want to clear the terminal.  Please provide a setting to disable this popup.  

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

Don't know which version changed this.

**Comment by rimrul** [comment]
> Don't know which version changed this.

This change was introduced as part of MinTTY 3.6.2, which was first included wit Git for Windows 2.39.0.

Add the following line to your `~/.minttyrc` file to restore the previous Alt+F8 hotkey behavior (reset the terminal with no confirmation dialog):

 `KeyFunctions=A+F8:reset-noask`

See https://github.com/mintty/mintty/issues/1173 for details.

**Comment by grosch-intl** [comment]
Thank you!
